File: Taller Tipos de clases/Go/Clase Interfaz/Pago.go
interface Pago {
    boolean procesar(double monto);
}

class PayPal implements Pago {
    String cuenta;

    public PayPal() {
        cuenta = "NA";
    }

    public PayPal(String cuenta) {
        this.cuenta = cuenta;
    }

    public PayPal(PayPal p) {
        this.cuenta = p.cuenta;
    }

    @Override
    public boolean procesar(double monto) {
        return true;
    }
}

File: Taller Tipos de clases/C#/Clase Interfaz/Pago.cs
interface IPago
{
    bool Procesar(double monto);
}

class PayPal : IPago
{
    public string Cuenta;

    public PayPal()
    {
        Cuenta = "NA";
    }

    public PayPal(string cuenta)
    {
        Cuenta = cuenta;
    }

    public PayPal(PayPal p)
    {
        Cuenta = p.Cuenta;
    }

    public bool Procesar(double monto)
    {
        return true;
    }
}

